ports/stm32f4/supervisor/port.c

```diff
@@ -58,6 +59,7 @@ void reset_port(void) {
 	reset_all_pins();
```

Comment:
indentation: use spaces, not tab

Reply:
I need to stop using sublime, it keeps sneaking these in